HW02_B_Mimic.md

```diff
@@ -37,6 +37,17 @@ data("diamonds")
 Using the diamonds dataset, make this graph:
 ![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-2-1.png)<!-- -->
 
+```{r graph1 code, echo=FALSE}
+ggplot(diamonds, aes(x = cut, fill = clarity)) + 
+  geom_bar(position = "dodge") +
+  xlab("Diamond Cut") +
+  ylab("Number of Diamonds") +
+  labs(title = "My Diamond Collection", subtitle = "Boxplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarity of diamond") +
+  theme(plot.title = element_text(hjust = 0.5), panel.background = element_rect(linetype = "solid", color = "black")) +

```

Comment:
I hadn't noticed the panel background black outline until reading it in your code, but now that I compare it with what I had (not including this part), I definitely see the difference.